src/test/resources/com/conveyal/datatools/gtfs/fake-agency-with-flex-version-1/translations.txt

```diff
@@ -0,0 +1,2 @@
+table_name,field_name,language,translation,record_id,record_sub_id,field_value
+stops,stop_desc,FR,en direction du nord,4u6g,,
```

Comment:
Do we need this file? The stops table does not have values populated for `stop_desc`.

Reply:
Updated the stop description to include a value.